src/granite/select.cr

```diff
@@ -0,0 +1,21 @@
+module Granite::Select
+  struct Container
```

Comment:
Could this be called a fieldset?

Reply:
It's not so much a set of fields as it is a (loose) container for the query (or SELECT) statement.  `Granite::Query` is used in your new query builder, so I tried to capture this as simply as possible.  I don't love `Granite::Select::Container` though.  Open to ideas.

Reply:
Sure `Container` is a little vague, but is it going to show up in an Amber devs code? If not, I don't think it's important to be 100% on target for merging this.

Reply:
Makes sense.  No, the only dev interface is the `query` macro, so not a big deal here.